DBM-GUI/modules/MainFrame.lua

```diff
@@ -66,10 +59,24 @@ frame:SetScript("OnMouseUp", function(self)
 end)
 frame.tabs = {}
 
-local frameResize = CreateFrame("Frame", nil, frame)
-frameResize:SetSize(10, 10)
-frameResize:SetPoint("BOTTOMRIGHT", frame, "BOTTOMRIGHT")
+if not isRetail then
+	CreateFrame("Button", "$parentClosePanelButton", frame, "UIPanelCloseButtonDefaultAnchors")
+	local titleBg = frame:CreateTexture("$parentTitleBga", "BACKGROUND", "_UI-Frame-TitleTileBg")
+	titleBg:ClearAllPoints()
+	titleBg:SetPoint("TOPLEFT", 8, -3)
+	titleBg:SetPoint("TOPRIGHT", -24, -3)
+end
+
+local frameBg = frame:CreateTexture("$parentBg", "BACKGROUND")
+frameBg:SetTexture("Interface\\FrameGeneral\\UI-Background-Rock")
+frameBg:SetColorTexture(0, 0, 0, 0.8)
+frameBg:SetPoint("TOPLEFT", 5, -21)
+frameBg:SetPoint("BOTTOMRIGHT", -2, isRetail and 2 or 8)
+
+local frameResize = CreateFrame("Button", nil, frame)
+frameResize:SetSize(25, 25)
 frameResize:EnableMouse(true)
```

Comment:
Removing this line might fix the issue with the window randomly becoming very large when you press it. I'm 99% sure that was how I fixed the issue with my own addon

Reply:
Hm it might be that the second part of this fix was to create an overlay that moves the window instead of the window itself. iirc there's a weird interaction when clicking on an element to resize that cascades to a window that can be moved  https://github.com/linaori/wow-battleground-commander/blob/cf1b414e78a94dd8d3e7d832261239c18c9d571b/BattlegroundTools.lua#L225

Reply:
The setup currently:

```
Main frame
 - resizable
 - MouseDown event
 - children
   - resizer
     - MouseDown  event
```

What would probably fix it:
```
Main frame
 - resizable
 - children
   - move overlay
     - MouseDown event
   - resizer
     - MouseDown event  
```

Most simple setup would be in the image below. In theory you can also use 2 frames that don't overlap with each other, but fill in a bit more space on the bottom/right, but is also more frames while it probably won't bother users they can't drag the edges there
![image](https://github.com/DeadlyBossMods/DBM-Unified/assets/1754678/4924991f-4e34-4788-8807-631dcda4aa9a)